conformance/conformance_test.cc

```diff
@@ -571,43 +606,43 @@ void ConformanceTestSuite::TestPrematureEOFForType(FieldDescriptor::Type type) {
 
   ExpectParseFailureForProto(
       tag(field->number(), wire_type),
-      "PrematureEofBeforeKnownNonRepeatedValue" + type_name, REQUIRED);
+      "PrematureEofBeforeKnownNonRepeatedValue" + type_name, REQUIRED, true);
```

Comment:
ExpectParseFailureForProto() should not take a boolean. It should test against both proto2 and proto3.